styles/styles.css

```diff
@@ -0,0 +1,88 @@
+.flex-container {
+  display: flex;
+  flex-direction: column;
+}
+.flex-container > div {
+  margin: 10px;
+  padding: 20px;
+  font-style: normal;
+}
+h1 {
+  font-weight: bold;
+  color: red;
+}
+h2 {
+  font-weight: bold;
+  color: red;
+  font-style: italic;
+}
+.header {
+  margin: auto;
+}
+.navbar > ul li {
+  color: orange;
+  list-style: none;
+  text-align: center;
+  padding: 30px 15px;
+  display: block;
+}
+.navbar li {
+  float: left;
```

Comment:
don't use floats for this. there are display properties that are better to display things in a line